masu/__init__.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""App factory for Masu application."""
+import logging
+import os
+
+from flask import Flask
+from flask.logging import default_handler
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
Okay. Let's find time this week to discuss.

My thinking is go as light as possible here. I know things tend to grow, but this realistically should have a status endpoint, and an endpoint to receive a message that new data is available--that endpoint will have to grow a little as we receive data from different sources. The rest is just pure Python processing. The only reason I see to use SQLAlchemy here is convenient session handling tied to requests. Otherwise I would use psycopg2. Once we're inserting data, we want to do so _en masse_, and we'll need to drop down to lower level than ORM for that to work.